src/main/java/org/folio/inventory/resources/TenantApi.java

```diff
@@ -0,0 +1,67 @@
+package org.folio.inventory.resources;
+
+import io.netty.handler.codec.http.HttpResponseStatus;
+import io.vertx.core.Future;
+import io.vertx.core.MultiMap;
+import io.vertx.core.Vertx;
+import io.vertx.core.logging.Logger;
+import io.vertx.core.logging.LoggerFactory;
+import io.vertx.ext.web.Router;
+import io.vertx.ext.web.RoutingContext;
+import org.folio.rest.tools.utils.TenantTool;
+import org.folio.rest.util.OkapiConnectionParams;
+import org.folio.util.pubsub.PubSubClientUtils;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class TenantApi {
+
+  private static final Logger LOG = LoggerFactory.getLogger(TenantApi.class);
+  private static final String TENANT_API_PATH = "/_/tenant";
+  private static final String OKAPI_TENANT_HEADER = "x-okapi-tenant";
+
+  public void register(Router router) {
+    router.post(TENANT_API_PATH).handler(this::postTenant);
+  }
+
+  private void postTenant(RoutingContext routingContext) {
+    MultiMap headersMap = routingContext.request().headers();
+    String tenantId = TenantTool.calculateTenantId(headersMap.get(OKAPI_TENANT_HEADER));
+    LOG.info("sending postTenant for " + tenantId);
+
+    Map<String, String> okapiHeaders = getOkapiHeaders(routingContext);
+    registerModuleToPubsub(okapiHeaders, routingContext.vertx())
+      .setHandler(ar ->
```

Comment:
Do we need exception handling here? `ar` can be either succeeded or failed future here.

Reply:
for now, we ignore the result of the module registration to mod-pubsub until mod-pubsub is used as a transport for the data import process

Reply:
Ok, thanks for explaining it.